.github/workflows/ci-lambda.yml

```diff
@@ -0,0 +1,84 @@
+name: Automated Python Test Gate for Lambda Updates
+
+on:
+  push:
+    branches:
+      - master
+      - BFD-3524
+    paths:
+      - ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/*.py
```

Comment:
Would it be worth checking for any `*.py` file here? That way any new python modules will automatically included and we don't need to remember to keep this in sync.

Reply:
There are also ancillary files that we should probably be filtering on, like `requirements.txt` and `dev-requirements.txt`.

Perhaps something like:

```
- `**/*.py`
- `requirements.txt`
- `dev-requirements.txt`
```